packages/pipeline-editor/src/PipelineExportDialog.tsx

```diff
@@ -26,16 +26,8 @@ interface IFileTypeSelectProps {
   temporarilyDisablePythonDSLForKFPSpecificPipelines?: boolean;
 }
 
-const FileTypeSelect: React.FC<IFileTypeSelectProps> = ({
-  fileTypes,
-  temporarilyDisablePythonDSLForKFPSpecificPipelines
-}) => {
-  // TODO: remove temporary workaround for KFP Python DSL export option
-  // See https://github.com/elyra-ai/elyra/issues/1760 for context.
+const FileTypeSelect: React.FC<IFileTypeSelectProps> = ({ fileTypes }) => {
   const _fileTypes = fileTypes.filter(t => {
```

Comment:
I believe this variable can be removed so that we use the `fileTypes` prop directly in the rendered component returned below (line 44).

Reply:
We can also remove lines 25-26
```
  // TODO: remove this prop
  temporarilyDisablePythonDSLForKFPSpecificPipelines?: boolean;
```

Reply:
Ah I see! Thanks for checking this out because this is the exact type of feedback I need for the frontend portion of this